rust/chains/hyperlane-sealevel/src/merkle_tree_hook.rs

```diff
@@ -85,17 +85,20 @@ impl Indexer<MerkleTreeInsertion> for SealevelMerkleTreeHookIndexer {
         &self,
         _range: RangeInclusive<u32>,
     ) -> ChainResult<Vec<(MerkleTreeInsertion, LogMeta)>> {
+        debug!("sealevel indexer not implemented for merkle tree hook");
```

Comment:
Is sealevel supposed to work atm? iiuc it shouldn't work and we're planning to implement this stuff in https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/2754 ?